docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -51,17 +51,37 @@ ReactDOM.render(
 
 You will need to host the custom checkout file online so it can be served by the store. You can use a hosting service such as [Amazon S3](https://aws.amazon.com/s3/). Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
 
-### Using WebDav to host a custom checkout
+### Using WebDAV to host a custom checkout
```

Comment:
@traci707, thanks for updating the spelling here. Be sure to add this to the [BigCommerce Glossary](https://intranet.bigcommerce.com/display/SST/BigCommerce+Glossary#BigCommerceGlossary-W) in our style guide.

Reply:
Will do